### DISCUSSION TITLE
Add copy button to file.php:linenumber

### DISCUSSION THREAD

**Discussion by BertvanHoekelen** [spatie/flareapp.io-roadmap#47] [Ideas]
It would be convenient to have a copy button next to file name and line number.   <br><br>

This should copy `app/App/Models/User.php:56`, so that you can browse to the correct file and line number in e.g. phpstorm.
<img width="496" alt="Screen Shot 2021-08-11 at 7 09 33 PM" src="https://user-images.githubusercontent.com/7521173/129072907-60398e91-e408-49dd-8952-aad3ee7af07c.png">

**Comment by AlexVanderbist** [comment]
Soon you'll be able to open paths in the report in PhpStorm, directly from Flare!
